utils/litemap/src/store/mod.rs

```diff
@@ -84,8 +89,6 @@ pub trait StoreMut<K, V>: Store<K, V> {
 
     /// Gets a key/value pair at the specified index, with a mutable value.
     fn lm_get_mut(&mut self, index: usize) -> Option<(&K, &mut V)>;
-    /// Pushes one additional item onto the store.
```

Comment:
Removing this from `StoreMut` does not allow implementing anything that couldn't be implemented otherwise.